### ISSUE TITLE
Translation for Geoblocker App

### ISSUE THREAD

**Issue by HomeITAdmin** [nextcloud/docker-ci#201]
Hello,

I followed the instruction from here to be able to add tanslation for the Geoblocker App for Nextcloud: 
[Adding Translations](https://docs.nextcloud.com/server/latest/developer_manual/app/view/l10n.html#adding-translations)

But nothing seems to reach Transifex, so obviously I am missing something or doing something wrong. Any hint where to start looking?

[Geoblocker App](https://github.com/HomeITAdmin/nextcloud_geoblocker)
[Issue](https://github.com/HomeITAdmin/nextcloud_geoblocker/issues/2)

Thanks

**Comment by skjnldsv** [comment]
cc @rullzer @blizzz

**Comment by rullzer** [comment]
A git pull was missing
should show up this evening.

**Comment by rullzer** [comment]
And it shows up on transifex

**Comment by HomeITAdmin** [comment]
@rullzer Thanks a lot. 
For me to learn, did I do something wrong? If yes, when and where ;-) ?

**Comment by rullzer** [comment]
> @rullzer Thanks a lot.
> For me to learn, did I do something wrong? If yes, when and where ;-) ?

No, you didn't. For some reason the config file did not get deployed.
Something else. Could you remove the nextcloud-bot as a collaborator and add it again. Seems I missed the e-mail to accept the invite on that account as well.

**Comment by HomeITAdmin** [comment]
Ok, I removed the nextcloud-bot and added it again.

**Comment by rullzer** [comment]
Ok it now has access.
Can you give it additional access? Right now it doesn't have permission to push to master as it is an protected branch.

**Comment by HomeITAdmin** [comment]
With a user account there is only one access level according to the documentation of Github.

I only activated the branch protection because it is recommended in the documentation [Adding Translation](https://docs.nextcloud.com/server/latest/developer_manual/app/view/l10n.html#adding-translations). I played a little bit around with it and activated "Require pull request reviews before merging". Could this be the problem? I deactivated it for now.
